webapp/sources/rudder/rudder-rest/pom.xml

```diff
@@ -98,6 +98,11 @@ along with Rudder.  If not, see <http://www.gnu.org/licenses/>.
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>org.springframework.security</groupId>
```

Comment:
I think this is used By RUdderProperties but i do feel we shouldn't have RudderProperties in Rudder-rest only in Rudder-web

Reply:
@ElaadF is it compiling without that dependency ?

Reply:
ok it's a dependency used in CurrentUserService

Reply:
It should be remvoed from rudder-web/pom.xml (if no compilation problem)